### DISCUSSION TITLE
Estrutura de diretórios no Github pages

### DISCUSSION THREAD

**Discussion by OliveiraJPE** [frontendbr/forum#2407] [Discussão]
Não consegui configurar a página com HTML, css, js com a estrutura de pasta. Só consegui subir colocando todos os arquivos no diretório raiz. Alguém sabe o pulo do gato.

**Comment by Diego-Girao** [comment]
Olá, se for o que eu entendi, você pode criar o diretório localmente onde você quiser e depois de linkar ele com o seu repo remoto aqui do github, você pode fazer o push(subir os arquivos), se você criou esse diretório na raiz(C:\>) e conectou ele com seu repo remoto, de fato só conseguirá fazer o push(subir os arquivos) de dentro dele mesmo. Vou anexar um print do diretório que to usando agora.

![image](https://user-images.githubusercontent.com/70491871/234841215-e306e81f-8292-404f-a633-088bce7e73c7.png)

**Comment by ninetails** [comment]
Nas configurações do repositório, em Pages, você consegue selecionar a _branch_ e a pasta que será utilizada no _deploy_

![configurações de Pages](https://user-images.githubusercontent.com/40065/234939194-2ff93cee-2604-4858-9227-dc2cdc1a82b0.png)

Importante que esteja tudo nela que você precise para seu site estático rodar: _assets_ como imagens, arquivos js, arquivos HTML, css.

Caso o repositório não seja seu `username.github.io`, bom também atentar que vc vai precisar adicionar o nome do repositório pois ele será criado como `username.github.io/nome-do-repositorio/`, provavelmente tendo que mudar também o caminho de chamada dos arquivos de _assets_.

**Comment by Marcia-Moreira** [comment]
Oi, o meu problema é mais básico acho!  Meu projeto estava organizado em pastas, template, statics, css, js, img... subi pro github, de boa. Mas quando tentei fazer o deploy ele só "renderizou" o readme, ignorou tudo que estava em pastas, já tirei o index e coloquei na raiz, mas ele não reconhece o css, js... Será que vou ter que deixar tudo solto na raiz do repositório? Tem outro jeito? as fotos soltas é o que mais me incomoda.

**Reply by AlexsandroSA** [reply]
Olá @Marcia-Moreira, tudo. bem?

Poderia enviar o repo,  encontrei [esse](https://github.com/Marcia-Moreira/marcia-moreira.github.io) e [esse](https://github.com/Marcia-Moreira/marciamoreira.github.io) em seu git ambos e só tem o README.md.

Como sugestão, adiciona um `index.html` simples, faz o push. Tem um passo a passo aqui https://pages.github.com/

Espero ter ajudado :)
Qualquer coisa ou dúvida só falar :)
